tests/src/Functional/NewsIntegrationTest.php

```diff
@@ -79,7 +79,7 @@ public function testCreateNews() {
     $page->fillField('Introduction', 'Example Introduction');
     $page->fillField('Date', '2022-01-24');
     $media_name = $media_image->getName() . ' (' . $media_image->id() . ')';
-    $page->fillField('Use existing media', $media_name);
+    $page->fillField('Media item', $media_name);
```

Comment:
Here we need to fill also the caption field, which this new "featured media" field type has.

Reply:
Nice call, I did for Events as well.